src/standard-live-connect.ts

```diff
@@ -111,7 +111,9 @@ function _standardInitialization (liveConnectConfig: LiveConnectConfig, external
       resolve: resolver.resolve.bind(resolver),
       resolutionCallUrl: resolver.getUrl.bind(resolver),
       config: validLiveConnectConfig,
-      eventBus: eventBus
+      eventBus: eventBus,
+      storageHandler: storageHandler,
+      callHandler: callHandler,
```

Comment:
Why are you exposing these two?

Reply:
so that liveconnectbuilder can use them instead of the raw unwrapped handlers provided by users

Reply:
Discussed in call.